offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/products/MaintainProductsView.groovy

```diff
@@ -0,0 +1,24 @@
+package life.qbic.portal.offermanager.components.products
+
+import com.vaadin.ui.FormLayout
+
+/**
+ *
+ * <h1>This class generates a Form Layout in which the user can maintain the service products</h1>
+ *
+ * <p>{@link MaintainProductsViewModel} will be integrated into the qOffer 2.0 Portlet and provides an User Interface
+ * with the intention of enabling an {@value life.qbic.portal.offermanager.security.Role#OFFER_ADMIN} to create, archive and copy products.</p>
+ *
+ * @since 1.0.0
+ *
+ */
+class MaintainProductsView extends FormLayout{
```

Comment:
Is this still correct according to #392 this View extends a VerticalLayout
```suggestion
class MaintainProductsView extends VerticalLayout{
```

Reply:
I think this is the responsibility of #392 and stay as is. It will get resolved when the other PR is merged